### DISCUSSION TITLE
Taskbar shifts down every time on using secondary monitor

### DISCUSSION THREAD

**Discussion by zavedlkce** [valinet/ExplorerPatcher#3995] [Q&A]
The taskbar gets shifts down every time I hook up secondary display using HDMI/type-C cable or the system goes to sleep.  I have to restart explorer to fix this issue. what could be the reason for this behaviour? Is there a fix available to it?
<img width="1919" alt="image" src="https://github.com/user-attachments/assets/d6ceed00-8352-494c-8f48-15c07f7c81af">

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.  Please include if you are running x86-64 or Arm.  And what language your windows is in.
Please post explorer patcher version.

Did you set your taskbar to be windows 10 (ExplorerPatcher) taskbar?
